sdks/python/test-suites/dataflow/py37/build.gradle

```diff
@@ -24,19 +24,12 @@ dependencies {
   distTarBall project(path: ":sdks:python", configuration: "distTarBall")
 }
 
-// Required to setup a Python 3 virtualenv.
+// Required to setup a Python 3 virtualenv and task names.
 pythonVersion = '3.7'
-
+apply from: "../common.gradle"
 
 def runScriptsDir = "${rootDir}/sdks/python/scripts"
 
-// Basic test options for ITs running on Jenkins.
-def basicTestOpts = [
-    "--nocapture",  // print stdout instantly
-    "--processes=8",  // run tests in parallel
-    "--process-timeout=4500", // timeout of whole command execution
-]
-
 def preCommitIT(String runScriptsDir, String envdir, Boolean streaming, Boolean runnerV2) {
```

Comment:
We can move preCommitIT out as well, in a separate change. This is an example of a test that we currently run against only one of high priority versions.